packages/react/src/patterns/blocks/ContentBlockMixedGroups/__stories__/ContentBlockMixedGroups.stories.js

```diff
@@ -0,0 +1,158 @@
+
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.
+      `;
+
+    const pictogramHeading =
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+    const pictogramItems = [
+      {
+        heading: 'Aliquam condimentum interdum',
+        copy:
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.',
+        cta: {
+          cta: {
+            href: 'https://www.example.com',
+          },
+          type: 'local',
+          copy: 'Lorem ipsum dolor',
+        },
+        pictogram: {
+          src: Desktop,
+          'aria-label': 'Desktop',
+        },
+      },
+      {
+        heading: 'Aliquam condimentum interdum',
+        copy:
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.',
+        cta: {
+          cta: {
+            href: 'https://www.example.com',
+          },
+          type: 'local',
+          copy: 'Lorem ipsum dolor',
+        },
+        pictogram: {
+          src: Pattern,
+          'aria-label': 'Pattern',
+        },
+      },
+      {
+        heading: 'Aliquam condimentum interdum',
+        copy:
+          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.',
+        cta: {
+          cta: {
+            href: 'https://www.example.com',
+          },
+          type: 'local',
+          copy: 'Lorem ipsum dolor',
+        },
+        pictogram: {
+          src: Touch,
+          'aria-label': 'Touch',
+        },
+      },
+    ];
+
+    const simpleHeading = ContentGroupSimpleKnobs.heading;
+    const simpleMediaData = ContentGroupSimpleKnobs.mediaData;
+    const simpleTypes = ContentGroupSimpleKnobs.types;
+    const simpleMediaType = simpleTypes.image;
+    const simpleItems = ContentGroupSimpleKnobs.items;
+    const simpleCta = ContentGroupSimpleKnobs.cta;
+
+    const items = [
+      {
+        type: 'ContentGroupCards',
+        heading: ContentGroupCardsKnobs.heading,
+        items: ContentGroupCardsKnobs.items,
+      },
+      {
+        type: 'ContentGroupPictograms',
+        heading: pictogramHeading,
+        items: pictogramItems,
+      },
+      {
+        type: 'ContentGroupSimple',
+        mediaType: simpleMediaType,
+        mediaData: simpleMediaData,
+        heading: simpleHeading,
+        items: simpleItems,
+        cta: simpleCta,
+      },
```

Comment:
Could we just limit to 3 by default? In the knobs maybe we can add a number of groups, and a select for each group so they choose and move around?

```javascript
const groupOptions = {
  'ContentBlockSimple': contentBlockSimpleData,
  'ContentBlockPictograms': contentBlockPictogramsData,
  'ContentBlockCards': contentBlockCardsData,
};
const numGroups = number("Number of groups", 1);
const items = [];

for (let = 0; i < numGroups; i++) {
  items.push(select(`Group #{i}`, groupOptions, groupOptions.ContentBlockSimple));
}

return (
  <ContentBlockMixedGroups
    heading={heading}
    copy={copy}
    items={items}
    cta={cta}
  />
)
```

Reply:
Will include one of each, but saving knobs for next sprint. We need to go through all these patterns and set the knobs consistently.